shm/test.cpp

```diff
@@ -0,0 +1,45 @@
+#include "cargo.hpp"
+#include "coordinates.hpp"
+#include "island.hpp"
+#include "map.hpp"
+#include "ship.hpp"
+#include "player.hpp"
+
+#include <vector>
+#include "gtest/gtest.h"
```

Comment:
Here you have headers from project, then in another paragraph standard library include with another header.

Reply:
Thanks, we will correct includes in next PR